delegation-toolkit/get-started/delegation-quickstart.md

```diff
@@ -62,11 +69,15 @@ const delegatorSmartAccount = await toMetaMaskSmartAccount({
 })
 ```
 
+:::note
+See [how to configure other smart account types](../how-to/create-smart-account/configure-accounts-signers.md).
+:::
+
 ### 4. Create a delegate account
 
-Create a delegate account to receive the delegation. The delegate can be either a smart account or an externally owned account (EOA).
+Create an account to represent Bob, the delegate who will receive the delegation. The delegate can be a smart account or an externally owned account (EOA).
```

Comment:
Maybe in the last step, we can have a note. If you wish to EOA as a delegate, See how to redeem the delegation with EOA? 
https://docs.metamask.io/delegation-toolkit/how-to/redeem-delegation/#redeem-with-an-eoa